### PR TITLE
Make CapacityMixin and use it for Workshop, Food and Meal model.

### PR DESCRIPTION
**Pull request by PetrDlouhy**
This is here to demonstrate, how I will handle capacity and how it will be watched over. I will implement this also for other models with capacity.

CapacityMixin holds also methods for determining if capacity is exceeded.
This is tested at `Workshop`, `Food` and `Meal` model.

### CODE REVIEW

File: api/tests/models/test_meal.py [hunk 1]
```diff
@@ -10,3 +17,39 @@ def test_string_representation(self):
         """Test that meal turns to string properly."""
         entry = Meal(name="Foo Meal")
         self.assertEqual(str(entry), 'Foo Meal')
+
+    @freeze_time("2017-2-1")
```
Comment (just-paja) [inline api/tests/models/test_meal.py]: We should use ISO-8601 for date formats by default.

**Comment by PetrDlouhy** [comment]
@just-paja Is the time definition now OK?

**Comment by just-paja** [comment]
@PetrDlouhy, depends. I thought you need only date, that's just "2017-02-01". But if you **need** time, then "2017-02-01T22:57:12+01:00" is ok.

**Comment by PetrDlouhy** [comment]
@just-paja  No, time is not needed. Now it should be fixed.

**Pull request merged**